src/components/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,88 @@
+        <script src="jquery-2.2.1.min.js"></script>
+        <script src="jquery.fabric.min.js"></script>
+    </head>
+    ```
+2. Copy the HTML from the sample above into your page. For example:
+    ```
+    <div class="ms-Breadcrumb">
+        <div class="ms-Breadcrumb-overflow">
+            <div class="ms-Breadcrumb-overflowButton ms-Icon ms-Icon--ellipsis" tabindex="1"></div>
+            <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            <div class="ms-Breadcrumb-overflowMenu">
+                <ul class="ms-ContextualMenu is-open"></ul>
+            </div>
+        </div>
+        <ul class="ms-Breadcrumb-list">
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="2">Files</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="3">Folder 1</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="4">Folder 2</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="5">Folder 3</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="6">Folder 4</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+            <li class="ms-Breadcrumb-listItem">
+                <a class="ms-Breadcrumb-itemLink" href="#" tabindex="7">Folder 5</a>
+                <i class="ms-Breadcrumb-chevron ms-Icon ms-Icon--chevronRight"></i>
+            </li>
+        </ul>
+    </div>
+    ```
+3. Add the following `<script>` tag to your page, below the references to jQuery and Fabric's JS, to instantiate all Breadcrumbs on the page:
+    ```
+    <script>
+        $(document).ready(function() {
+            if (typeof fabric !== "undefined") {
+                if ('Breadcrumb' in fabric) {
+                    var elements = document.querySelectorAll('.ms-Breadcrumb');
+                    var i = elements.length;
+                    var component;
+                    while(i--) {
+                        component = new fabric['Breadcrumb'](elements[i]);
+                    }
+                }
+            }
+        });
+    </script>
+    ```
+4. Verify that the component is working the same as in the sample above.
+5. Replace the sample HTML content (such as the `.ms-Breadcrumb-listItem` elements) with your content.
+
+## Dependencies
```

Comment:
Yes, exactly. Soon we shouldn't have any dependencies on jQuery so this will be less confusing.